jobs/build/coreos-installer_sync/Jenkinsfile

```diff
@@ -83,34 +101,42 @@ node {
         )
     }
 
-    stage("Sync to mirror") {
-        def mirror = "use-mirror-upload.ops.rhcloud.com"
-        def dir = "/srv/pub/openshift-v4/x86_64/clients/coreos-installer"
+    stage("calculate shasum") {
+        commonlib.shell(
+            script: "cd ${workdir} && sha256sum * > sha256sum.txt",
+        )
+    }
 
+    stage("Sync to mirror") {
         if (params.DRY_RUN) {
             commonlib.shell(
                 script: [
                     "echo 'Would have synced the following to ${mirror}:'",
                     "tree ${workdir}/${params.VERSION}",
                 ].join('\n')
             )
-            return
+        } else {
+            sh "tree ${workdir} && cat ${workdir}/sha256sum.txt"
+            commonlib.syncDirToS3Mirror("${workdir}/${params.VERSION}/", "/pub/openshift-v4/x86_64/clients/coreos-installer/${params.VERSION}/")
+            commonlib.syncDirToS3Mirror("${workdir}/${params.VERSION}/", "/pub/openshift-v4/x86_64/clients/coreos-installer/latest/")
         }
+    }
 
-        commonlib.syncDirToS3Mirror("${workdir}/${params.VERSION}/", "/pub/openshift-v4/x86_64/clients/coreos-installer/${params.VERSION}/")
-        commonlib.syncDirToS3Mirror("${workdir}/${params.VERSION}/", "/pub/openshift-v4/x86_64/clients/coreos-installer/latest/")
-
-        sshagent(['aos-cd-test']) {
-            commonlib.shell(
-                script: """
-                    set -euxo pipefail
-                    cd ${workdir}
-                    ssh ${mirror} mkdir -p ${dir}
-                    scp -r ${params.VERSION} ${mirror}:${dir}/${params.VERSION}
-                    ssh ${mirror} ln --symbolic --force --no-dereference ${params.VERSION} ${dir}/latest
-                    ssh ${mirror} tree ${dir}
-                    ssh ${mirror} /usr/local/bin/push.pub.sh openshift-v4/x86_64/clients/coreos-installer -v
-                """
+    stage("sign artifacts") {
+        if (params.DRY_RUN) {
+            echo "Would have signed artifacts"
+            return
+        } else {
+            release.signArtifacts(
+                name: params.VERSION,
+                signature_name: "signature-1",
+                dry_run: params.DRY_RUN,
+                env: params.ENV,
+                key_name: params.KEY_NAME,
+                arch: "x86_64",
+                digest: "",
+                client_type: "",
```

Comment:
Not sure what this is or should be, but don't think this should be empty.

Reply:
same as above